src/Stream/BidirectionalReader.h

```diff
@@ -7,6 +7,18 @@ namespace OP2Utility::Stream
 {
 	class BidirectionalReader : public ForwardReader {
 	public:
+		void Peek(void* buffer, std::size_t size) {
+			ReadImplementation(buffer, size);
+			SeekBackward(size);
+		}
+
+		// Trivially copyable data types
+		template<typename T>
+		std::enable_if_t<std::is_trivially_copyable<T>::value>
+			Peek(T& object) {
+			Peek(&object, sizeof(object));
+		}
```

Comment:
Something I just noticed post-merge. The indentation for the `Peek` method is a bit odd. The function signature is indented the same amount as the function implementation. I think when I initially read this, I interpreted it as a 2 line function body.

Reply:
Yeah good catch. I'll fix it this weekend.